tests/test_local.py

```diff
@@ -153,3 +156,113 @@ def test_criteria():
+        )
+        expected = [1.16666667, 1.61538462, 2.625, 7.0]
+        result = _get_plotting_positions(data)
+        np.testing.assert_array_almost_equal(result.streamflow_pp, expected)
+        np.testing.assert_array_almost_equal(result.streamflow, data.streamflow)
+
+        data_2d = xr.concat([data, data], dim="id")
+        result = _get_plotting_positions(data_2d)
+        np.testing.assert_array_almost_equal(result.streamflow_pp, [expected, expected])
+        np.testing.assert_array_equal(result.streamflow, data_2d.streamflow)
+
+    def test_nan(self):
+        data = timeseries(
+            np.array([50, np.nan, 80, 95]),
+            variable="streamflow",
+            start="2001-01-01",
+            freq="YS",
+            as_dataset=True,
+        )
+        expected = [1.23076923, np.nan, 2.0, 5.33333333]
+        result = _get_plotting_positions(data)
+        np.testing.assert_array_almost_equal(result.streamflow_pp, expected)
+        np.testing.assert_array_almost_equal(result.streamflow, data.streamflow)
+
+        data_2d = xr.concat([data, data], dim="id")
+        result = _get_plotting_positions(data_2d)
+        np.testing.assert_array_almost_equal(result.streamflow_pp, [expected, expected])
+        np.testing.assert_array_equal(result.streamflow, data_2d.streamflow)
+
+    def test_return_period(self):
+        data = timeseries(
+            np.array([50, 65, 80, 95]),
+            variable="streamflow",
+            start="2001-01-01",
+            freq="YS",
+            as_dataset=True,
+        )
+        expected = [0.14285714, 0.38095238, 0.61904762, 0.85714286]
+        result = _get_plotting_positions(data, return_period=False)
+        np.testing.assert_array_almost_equal(result.streamflow_pp, expected)
+        np.testing.assert_array_almost_equal(result.streamflow, data.streamflow)
+
+        data_2d = xr.concat([data, data], dim="id")
+        result = _get_plotting_positions(data_2d, return_period=False)
+        np.testing.assert_array_almost_equal(result.streamflow_pp, [expected, expected])
+        np.testing.assert_array_equal(result.streamflow, data_2d.streamflow)
+
+    def test_alpha_beta(self):
+        data = timeseries(
+            np.array([50, 65, 80, 95]),
+            variable="streamflow",
+            start="2001-01-01",
+            freq="YS",
+            as_dataset=True,
+        )
+        expected = [1.25, 1.66666667, 2.5, 5.0]
+        result = _get_plotting_positions(data, alpha=0, beta=0)
+        np.testing.assert_array_almost_equal(result.streamflow_pp, expected)
+        np.testing.assert_array_almost_equal(result.streamflow, data.streamflow)
+
+        data_2d = xr.concat([data, data], dim="id")
+        result = _get_plotting_positions(data_2d, alpha=0, beta=0)
+        np.testing.assert_array_almost_equal(result.streamflow_pp, [expected, expected])
+        np.testing.assert_array_equal(result.streamflow, data_2d.streamflow)
+
+
+class Testprepare_plots:
+
```

Comment:
```suggestion
ds = timeseries(
    np.array([50, 65, 80, 95, 110, 125, 140, 155, 170, 185, 200]),
    variable="streamflow",
    start="2001-01-01",
    freq="YS",
    as_dataset=True,
)
params = xhfa.local.fit(ds, distributions=["gamma", "pearson3"])

```
Je crois que tu pourrais faire ceci plutôt que d'ajouter la fonction à `testing.utils`. ça me semble plus clean (pour éviter d'importer xhydro dans `testing`).

Ensuite, tu peux simplement utiliser `self.params` dans tes tests pour cette classe.